wagtail/images/views/images.py

```diff
@@ -291,8 +292,11 @@ def preview(request, image_id, filter_spec):
     image = get_object_or_404(get_image_model(), id=image_id)
 
     try:
-        response = HttpResponse()
-        image = Filter(spec=filter_spec).run(image, response)
+        # Temporary image needs to be an instance that Willow can run optimizers on
+        temp_image = BytesIO()
```

Comment:
Issue: We should use `SpooledTemporaryFile` here to avoid needing to load the entire file into memory.

Reply:
👍 - copied max_size from https://github.com/wagtail/wagtail/blob/ac38343957787b7d7a2ff34164a98538e177ff96/wagtail/images/models.py#L771